### DISCUSSION TITLE
GitHub Compilation Method

### DISCUSSION THREAD

**Discussion by jamesohdi** [community/community#58434] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I would like to download this GitHub's 
https://github.com/planqi/slashdiablo-maphack 
corresponding source and use it with personal modifications.
But I'm too new to build.
I read the contents of the bottom part of the build, but I don't understand it.
So after installing the 2019 version of Visual Studio
I tried to build and compile, but I get too many errors.
I know I'm doing something wrong.
Which program should I use to compile,
Could you tell me the approximate method?

**Comment by airtower-luna** [comment]
There is no "GitHub compilation method", how to build depends on each project. So it's best to look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue or discussion there is usually good. :slightly_smiling_face:

From a glance at the repository it looks like that specific project uses CMake to build.
